onnxruntime/python/tools/transformers/models/llama/single_run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Let's rename this file to `single_run_70b_model.sh` so that it is clear this script is just for the 70B model.